libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tab.component.scss

```diff
@@ -218,12 +282,18 @@
     var(--sky-color-text-deemphasized)
   );
   pointer-events: none;
+
+  .sky-vertical-tab-right-arrow {
+    color: var(--sky-color-icon-deemphasized);
```

Comment:
does this only apply to modern somehow? Or is it missing an override? Actually, it looks like color is set on the exact same selector above but that one has an override, is this possibly duplicate?

Reply:
This style isn't really possible today. The arrow only shows on sectioned form buttons and they can not be disabled. I put the token in per the spec - but didn't put in an override because it isn't actually possible in default and modern today.

Reply:
should we just remove it then? or it will be possible in v2?